main.go

```diff
@@ -2,18 +2,29 @@ package main
 
 import (
 	"flag"
+	"fmt"
 	"log"
+	"os"
+	"path/filepath"
 
 	"github.com/fullstorydev/hauser/client"
 	"github.com/fullstorydev/hauser/config"
 	"github.com/fullstorydev/hauser/internal"
 	"github.com/fullstorydev/hauser/warehouse"
 )
 
+var version = "dev build <no version set>"
```

Comment:
Doh! I guess this means `make install` has never worked?

Sorry. That `Makefile` was literally copied from one of the other repos, so I must not have done enough to verify it...

Reply:
It's all good!